### PR TITLE
Re-order footer list

### PR DESCRIPTION
**Pull request by MarieDale3**
The footer list re-arranged to match Web and portal's new order

### CODE REVIEW

File: src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -157,13 +157,13 @@
                     <h2 class="govuk-visually-hidden">Support links</h2>
                     <ul class="govuk-footer__inline-list">
                         <li class="govuk-footer__inline-list-item">
-                            <a class="govuk-footer__link" asp-controller="Home" asp-action="TermsOfUse" asp-route-clientId="@ViewBag.ClientId">Terms of use</a>
+                            <a class="govuk-footer__link" asp-controller="Home" asp-action="Cookies" asp-route-clientId="@ViewBag.ClientId" target="_blank">Cookies</a>
```
Comment (benagain) [inline src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml:160]: SonarCloud suggests adding `noopener` and `noreferrer` relations to links that open in a new tab (related to [this security concern](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/))

```suggestion
                            <a class="govuk-footer__link" asp-controller="Home" asp-action="Cookies" asp-route-clientId="@ViewBag.ClientId" target="_blank" rel="noopener noreferrer">Cookies</a>
```

File: src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -157,13 +157,13 @@
...
                         </li>
                         <li class="govuk-footer__inline-list-item">
-                            <a class="govuk-footer__link" asp-controller="Home" asp-action="Cookies" asp-route-clientId="@ViewBag.ClientId">Cookies</a>
+                            <a class="govuk-footer__link" asp-controller="Home" asp-action="Privacy" asp-route-clientId="@ViewBag.ClientId" target="_blank">Privacy</a>
```
Comment (benagain) [inline src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml:163]:
```suggestion
                            <a class="govuk-footer__link" asp-controller="Home" asp-action="Privacy" asp-route-clientId="@ViewBag.ClientId" target="_blank" rel="noopener noreferrer">Privacy</a>
```

File: src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -172,14 +172,14 @@
                     </svg>
                     <span class="govuk-footer__licence-description">
                         All content is available under the
-                        <a class="govuk-footer__link" href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+                        <a class="govuk-footer__link" target="_blank" href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
```
Comment (benagain) [inline src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml:175]:
```suggestion
                        <a class="govuk-footer__link" target="_blank" rel="noopener noreferrer" href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
```

File: src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -172,14 +172,14 @@
...
                            rel="license">
                             Open Government Licence v3.0
                         </a>, except where otherwise stated
                     </span>
                 </div>
                 <div class="govuk-footer__meta-item">
-                    <a class="govuk-footer__link govuk-footer__copyright-logo" href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">
+                    <a class="govuk-footer__link govuk-footer__copyright-logo" target="_blank" href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">
```
Comment (benagain) [inline src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml:182]:
```suggestion
                    <a class="govuk-footer__link govuk-footer__copyright-logo" target="_blank" rel="noopener noreferrer" href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">
```

**Review by benagain — CHANGES_REQUESTED** [review]
Have the govuk-frontend files actually changed?  I couldn't see any differences.

**Comment by MarieDale3** [comment]
> Have the govuk-frontend files actually changed? I couldn't see any differences.

The logo link text wasn't underlined at the bottom for some reason, it should be underlined now

File: src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (MarieDale3) [reply src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml]: Thanks for this, I've amended the links

**Review by benagain — APPROVED** [review]

**Pull request merged**